### DISCUSSION TITLE
Question about changing github username

### DISCUSSION THREAD

**Discussion by pathetic** [community/community#18453] [Other Feature Feedback, Questions, & Ideas] [answered]
If i change my github username do my profile views reset?

**Comment by PaulDotSH** [comment, marked as answer]
Yes, they do reset.

**Reply by PaulDotSH** [reply]
> It's so obvious that you use a second account to answer your own questions joy

Did you even look at our profiles? People with nothing else to do can only hate :)
Maybe first you could code some projects, contribuite to something open-source

**Reply by PaulDotSH** [reply]
> It's so obvious that you use a second account to answer your own questions 😂
@Zerotask 

You still didnt code anything useful btw, keep on hating on others, here's you medal for not helping the society 🥇

**Reply by pathetic** [reply]
+1
@PaulDotSH leave him alone, he uses php 😛
